litmus-portal/frontend/src/components/ProgressBar/AnalyticsLinearProgressBar/index.tsx

```diff
@@ -13,17 +13,33 @@ const AnalyticsLinearProgressBar: React.FC<LinearProgressBarProps> = ({
   maxValue,
   isInTable,
 }) => {
-  const width: number = 2;
+  
   const resultValue = ((value as number) / (maxValue as number)) * 100;
 
-  const { palette } = useTheme();
+  const styleClass = useStyle();
```

Comment:
We follow the naming convention like this:
```suggestion
  const classes = useStyle();
```